client/src/pages/DetailsPage/item-details.jsx

```diff
@@ -0,0 +1,54 @@
+import '../DetailsPage/item-details.scss';
+import GoBackButton from "../../components/GoBackButton/GoBackButton";
+import React, { useState } from 'react';
+
+export const ItemDetails = (props) => {
+	const [quantity, setQuantity] = useState(0);
+
+	const incrementQuantity = () => {
+		setQuantity(quantity + 1);
```

Comment:
Funktioniert zwar. Es ist allerdings zu bevorzugen einen funktionsbasierten Aufruf zu wählen.
`setQuantity(previousQuantity => previousQuantity + 1)`
Das ist ein bisschen sicherer in der Funktionsweise.

Reply:
done